omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/LocationTagController1_8Test.java

```diff
@@ -109,17 +109,34 @@ public void shouldListAll() throws Exception {
 	}
 	
 	@Test
-	public void shouldUpdateLocationTag() throws Exception {
+	public void shouldUpdateLocationTagName() throws Exception {
 		
-		final String editedName = "Location Tag edited";
-		String json = "{ \"name\":\"" + editedName + "\" }";
+		final String EDITED_NAME = "Location Tag edited";
```

Comment:
Why don't you combine name and description update in the same ticket like you have been doing for others?

Reply:
> combine name and description update in the same test

good ideal and makes our code base precise.